### PR TITLE
ELEMENTS-765: Use variable to manage nuxeo-document-preview elements …

### PR DESCRIPTION
**Pull request by atoom75**
…min-height

### CODE REVIEW

**Comment by nmpcunha** [comment]
👍

File: nuxeo-document-preview.html [hunk 1]
```diff
@@ -52,12 +52,12 @@
       nuxeo-pdf-viewer {
         width: 100%;
         height: 100%;
-        min-height: 60vh;
+        min-height: var(--nuxeo-image-viewer-min-height, 60vh);
```
[the remark this answers is not in the document]
Comment (nmpcunha) [reply nuxeo-document-preview.html]: Nice catch!

[the remark this answers is not in the document]
Comment (atoom75) [reply nuxeo-document-preview.html]: Update made!

**Comment by Gabez0r** [comment]
👍

**Pull request merged**